app/ts/components/SendEthPage.tsx

```diff
@@ -48,40 +44,61 @@ const Main = () => {
 						<SendGuide />
 					</div>
 					<div class='[grid-area:controls]'>
-						<SendActions />
+						<FormActions />
 					</div>
 				</FormLayout>
 			</SendForm>
-		</TransferProvider>
+		</ErrorBoundary>
 	)
 }
 
+const SendForm = ({ children }: { children: ComponentChildren }) => {
+	const accountStore = useAccountStore()
+	const transfer = useTransferStore()
+
+	const handleSubmit = (event: Event) => {
+		event.preventDefault()
+
+		if (!accountStore.value.isConnected) {
+			accountStore.value.connectMutation.dispatch()
+			return
+		}
+
+		if (!transfer.value.isSigned) {
+			transfer.value.transactionResponseQuery.dispatch()
+			return
+		}
```

Comment:
It feels like we should tell the user about this rather than silently failing to submit the transaction.

Reply:
Since this is using the account store, Components using it responds to the error; submit button text says "unable to connect" and the header button shows an error. I am inclined to this as well there should be a better way to announce this to the user. created a separate issue a #65